src/components/Input.js

```diff
@@ -6,15 +6,15 @@ export class Input extends React.Component {
   render () {
     return (
       <input
-        placeholder={this.props.defaultValue}
+        placeholder="Program Name"
```

Comment:
this is still a code smell. What does the placeholder "Program Name" have to do with a generic Input component? If what you are trying to do is to just create a thin wrapper around the `<input>` element that sets some defaults, it would look something like this:

```javascript
render() {
  return (
    <input {...this.props}
           type="text"
           style={[
             styles.forms.base,
             styles.forms[this.props.kind],
             this.props.style
           ]} />
  );
}
```
With this you are effectively saying, "pass all the props right on through to <input>, and then override some of them with these specific ones (`type` and `style`)"